### ISSUE TITLE
the executable does not start without VS dlls

### ISSUE THREAD

**Issue by cirquit** [cirquit/trdrop#8]
This may be a problem because of the export type "debug", try "release"

**Comment by cirquit** [comment]
This is hopefully fixed now, Release Mode in VS works. Still needs testing.

**Comment by cirquit** [comment]
This is not fixed, we currently compile with the `/MTd` flag and have linker errors if we try to do `/MT`.
We may need to recompile `yaml-cpp` again with this flag.
```
1>libyaml-cppmdd.lib(parse.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(exceptions.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(convert.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(memory.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(node_data.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(parser.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(nodebuilder.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(directives.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(scanner.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(singledocparser.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(regex_yaml.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(stream.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(simplekey.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(scantoken.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(tag.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(null.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(scanscalar.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(scantag.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>libyaml-cppmdd.lib(exp.obj) : error LNK2038: mismatch detected for 'RuntimeLibrary': value 'MDd_DynamicDebug' doesn't match value 'MTd_StaticDebug' in main.obj
1>LINK : warning LNK4098: defaultlib 'MSVCRTD' conflicts with use of other libs; use /NODEFAULTLIB:library
1>libcpmtd.lib(locale.obj) : error LNK2001: unresolved external symbol _calloc_dbg
1>libcpmtd.lib(wlocale.obj) : error LNK2001: unresolved external symbol _calloc_dbg
1>libcpmtd.lib(xlocale.obj) : error LNK2001: unresolved external symbol _calloc_dbg
1>main.obj : error LNK2001: unresolved external symbol _calloc_dbg
1>libcpmtd.lib(cond.obj) : error LNK2001: unresolved external symbol _calloc_dbg
1>libcpmtd.lib(mutex.obj) : error LNK2001: unresolved external symbol _calloc_dbg
1>libcpmtd.lib(_tolower.obj) : error LNK2001: unresolved external symbol _calloc_dbg
1>libcpmtd.lib(xwcsxfrm.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(StlCompareStringA.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(locale.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(StlLCMapStringA.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(wlocale.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(xlocale.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>main.obj : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(cout.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(locale0.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libcpmtd.lib(cerr.obj) : error LNK2001: unresolved external symbol _malloc_dbg
1>libconcrtd.lib(location.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(UMSBackgroundPoller.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(UMSFreeVirtualProcessorRoot.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(FreeThreadProxy.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(ThreadProxy.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(UMSFreeThreadProxy.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(SearchAlgorithms.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(WorkQueue.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(Chores.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(VirtualProcessorRoot.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(FreeVirtualProcessorRoot.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(UMSThreadInternalContext.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(TransmogrifiedPrimary.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(Transmogrificator.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(TaskCollectionBase.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(CacheLocalScheduleGroup.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(FairScheduleGroup.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(ExternalContextBase.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(UMSThreadVirtualProcessor.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(SchedulingNode.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(SubAllocator.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(SchedulingRing.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(ScheduleGroupBase.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(WinRTWrapper.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(ThreadProxyFactoryManager.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(InternalContextBase.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(VirtualProcessor.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(SchedulerPolicyBase.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(ExecutionResource.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(HillClimbing.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(SchedulerProxy.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(UMSWrapper.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(UMSThreadProxy.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(ContextBase.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(TaskCollection.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(Platform.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(utils.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(ResourceManager.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(SchedulerBase.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(xmbtowc.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(StlCompareStringA.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(event.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libconcrtd.lib(rtlocks.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(locale.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(StlLCMapStringA.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(wlocale.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(xlocale.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(syserror.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(cerr.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(stdthrow.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(thread0.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>main.obj : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(taskscheduler.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(filesys.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(cout.obj) : error LNK2001: unresolved external symbol _CrtDbgReportW
1>libcpmtd.lib(wlocale.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(xlocale.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(xwcsxfrm.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(StlCompareStringA.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(cerr.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(mutex.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(locale.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(StlLCMapStringA.obj) : error LNK2001: unresolved external symbol _free_dbg
1>main.obj : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(cond.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(cout.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libcpmtd.lib(locale0.obj) : error LNK2001: unresolved external symbol _free_dbg
1>libyaml-cppmdd.lib(scantag.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(exp.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(simplekey.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(scantoken.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(tag.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(scanscalar.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(scanner.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(singledocparser.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(regex_yaml.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(stream.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(node_data.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(parser.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(nodebuilder.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(directives.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(parse.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(exceptions.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(convert.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(memory.obj) : error LNK2001: unresolved external symbol __imp__CrtDbgReportW
1>libyaml-cppmdd.lib(tag.obj) : error LNK2001: unresolved external symbol __imp__wassert
1>libyaml-cppmdd.lib(node_data.obj) : error LNK2001: unresolved external symbol __imp__wassert
1>libyaml-cppmdd.lib(nodebuilder.obj) : error LNK2001: unresolved external symbol __imp__wassert
1>libyaml-cppmdd.lib(scanner.obj) : error LNK2001: unresolved external symbol __imp__wassert
1>libyaml-cppmdd.lib(singledocparser.obj) : error LNK2001: unresolved external symbol __imp__wassert
1>libcpmtd.lib(_tolower.obj) : error LNK2001: unresolved external symbol _wcsdup_dbg
1>libcpmtd.lib(xstrcoll.obj) : error LNK2001: unresolved external symbol _wcsdup_dbg
1>libcpmtd.lib(locale.obj) : error LNK2001: unresolved external symbol _realloc_dbg
1>libconcrtd.lib(SubAllocator.obj) : error LNK2001: unresolved external symbol _CrtIsValidHeapPointer
1>C:\Users\rewrite\Documents\github-repos\trdrop\trdrop_c\x64\Release\trdrop_c.exe : fatal error LNK1120: 9 unresolved externals
1>Done building project "trdrop_c.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

**Comment by cirquit** [comment]
Resolved by 3c2e16bfb282715797f8399cfbb2b5da8e05963f
